Tools/GrafanaDashboard/grafanaalert.md

```diff
@@ -0,0 +1,35 @@
+# Grafana alert setup
+
+In this example we will create an low OEE alert for one of the production lines. First login to your Grafana service and select in the menu for Alert rules.
+
+![Navigate](navigatetoalerts.png)
+
+Then click on 'Create Rule' on the right.
+
+![Create rule](createrule.png)
+
+Then give your alert a name and select 'Azure Data Explorer' as data source. Click on query on the left
+
+![Alert query](alertquery.png)
+
+Put in the following in the query field and try to run your query. In this example we use the 'Seattle' production line. 
+
+```
+let oee = CalculateOEEForStation("assembly", "seattle", 6, 6);
+print round(oee * 100, 2)
+```
+and select 'table' as output. 
+
+Scroll down to the next section. In this example your will configure the alert threshold. In this example we will use below '10' as threshold, but in production environment this will be higher.
```

Comment:
Scroll down to the next section. Here, you will configure the alert threshold. In this example, we will use 'below 10' as the threshold, but in production environments, this will be higher.